lib/bolt/config.rb

```diff
@@ -214,6 +268,10 @@ def update_from_file(data)
         update_logs(data['log'])
       end
 
+      if data['plugins'].is_a?(Hash)
```

Comment:
Are any of these changes in `update_from_file` still needed? Since all we've changed now is the input data, it seems like this method should behave the same as it did before.